ui-admin/src/portal/siteContent/SiteContentEditor.tsx

```diff
@@ -251,30 +259,68 @@ const SiteContentEditor = (props: InitializedSiteContentViewProps) => {
             </Button>
           </h5>
         </div>
-        {
-          isEditable && <div className="d-flex flex-grow-1">
-            <Button className="ms-auto me-md-2" variant="primary"
-              disabled={readOnly || hasInvalidSection || (siteContent === workingContent)}
-              tooltipPlacement={'left'}
-              tooltip={(() => {
-                if (siteContent === workingContent) {
-                  return 'Site is unchanged. Make changes to save.'
-                }
-                if (hasInvalidSection) {
-                  return 'Site is invalid. Correct to save.'
-                }
-                return 'Save changes'
-              })()}
-              onClick={() => createNewVersion(workingContent)}>
+        <div className="d-flex flex-grow-1 justify-content-end">
+          {
+            isEditable && <>
+              <Button className="me-md-2" variant="primary"
+                disabled={readOnly || hasInvalidSection || (siteContent === workingContent)}
+                tooltipPlacement={'left'}
+                tooltip={(() => {
+                  if (siteContent === workingContent) {
+                    return 'Site is unchanged. Make changes to save.'
+                  }
+                  if (hasInvalidSection) {
+                    return 'Site is invalid. Correct to save.'
+                  }
+                  return 'Save changes'
+                })()}
+                onClick={() => createNewVersion(workingContent)}>
                   Save
-            </Button>
-            {
-            // eslint-disable-next-line
+              </Button>
+              {
+                // eslint-disable-next-line
             // @ts-ignore  Link to type also supports numbers for back operations
-              <Link className="btn btn-cancel" to={-1}>Cancel</Link>
-            }
+                <Link className="btn btn-cancel" to={-1}>Cancel</Link>
+              }
+            </>
+          }
+          <a className="btn" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Actions <FontAwesomeIcon className="ms-2" icon={faEllipsisV}/>
+          </a>
+          <div className="dropdown-menu">
+            <ul className="list-unstyled">
```

Comment:
```suggestion
            <ul className="list-unstyled mb-0">
```
I noticed that the `ul` adds some weird margin on the bottom